images/bootstrap/runner.sh

```diff
@@ -82,6 +82,7 @@ if [[ "${PODMAN_IN_CONTAINER_ENABLED}" == "true" ]]; then
                unix:///${XDG_RUNTIME_DIR}/podman/podman.sock \
                >/var/log/podman.log 2>&1 &
         echo "${!}" > /var/run/podman.pid
+	ln -s ${XDG_RUNTIME_DIR}/podman/podman.sock /var/run/docker.sock
```

Comment:
Btw I tried that with vm based providers, and it didn't work
nor will be needed soon when 
https://github.com/kubevirt/kubevirt/pull/8010
https://github.com/kubevirt/kubevirt/pull/8048
are merged

I think we better fix whatever needed and remove all softlink to docker

Reply:
I am not sure if you saw the prowjob linked in the commit - https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2191/rehearsal-check-provision-k8s-1.23/1544729115590922240#1:build-log.txt%3A2152 - its a panic from gocli and I don't see updates in this area in the changes you linked? 

Here is the PR for making gocli work with podman - https://github.com/kubevirt/kubevirtci/pull/554

Reply:
It looks to be using a few docker specific modules so it may not be as straight forward as the other scripts.
https://github.com/kubevirt/kubevirtci/blob/111fd507537476c5e5ff9d7a914c6ece8862aaaa/cluster-provision/gocli/docker/docker.go#L15-L17